cmd/traffic_crashlog/Makefile.am

```diff
@@ -20,6 +20,7 @@ bin_PROGRAMS = traffic_crashlog
 AM_CPPFLAGS = \
   $(iocore_include_dirs) \
   -I$(top_srcdir)/lib \
+  -I$(top_srcdir)/lib/ts \
```

Comment:
Well, Leif will go ballistic if this creeps back in, it was a very painful experience to take it out.